CHANGELOG-2.x.md

```diff
@@ -2,6 +2,7 @@
 
 ## 2.56.0 (not released yet, in development)
 
+* [MPFE-1017] Moment.js downgraded to v2.17.1 && added ability to change it from config file
```

Comment:
Please specify what bundle and what semantic configuration key needs to be used for that.